Fix wrong UOM in check_uptime windows plugin

mcktr opened the pull request:
This fixes the usage of invalid unit of measurement in the check_uptime windows plugin.

The performance data will now be provided without any UOM.

fixes #6406

dnsmichi commented:
I would opt for seconds, since this is the lowest UOM which fits here. The Graphite Writer feature also would just normalize this (tag support is pending), while future versions for InfluxDB will also send the UOM for better graph tagging.

```
s - seconds (also us, ms)
```

mcktr commented:
With the last commit the Performance Data output is always in seconds.

```
.\check_uptime.exe
UPTIME OK 16508s | uptime=16508s;;;0;

.\check_uptime.exe -u h
UPTIME OK 4h | uptime=16533s;;;0;

.\check_uptime.exe -u ms
UPTIME OK 16545515ms | uptime=16545s;;;0;

.\check_uptime.exe -w 1000 -c 5000
UPTIME CRITICAL 16566s | uptime=16566s;1000;5000;0;

.\check_uptime.exe -w 1000 -c 5000 -u h
UPTIME OK 4h | uptime=16581s;3600000;18000000;0;

.\check_uptime.exe -w 1000 -c 5000 -u ms
UPTIME CRITICAL 16594718ms | uptime=16594s;1;5;0;
```

dnsmichi commented:
Now the logic is duplicated into many source code locations. Wouldn't it be more reasonable to only adjust the printing of the performance data?

mcktr commented:
Agreed. I refactored the "convert to seconds" logic to be more generic and moved them into `thresholds.cpp`, thereby we can re-use this logic if we add another plugin which needs this.

dnsmichi commented:
```
C:\Users\michi\source\repos\icinga2\Bin\Debug\Debug>check_uptime.exe
UPTIME OK 37565s | 'uptime'=37565s;;;0;

C:\Users\michi\source\repos\icinga2\Bin\Debug\Debug>check_uptime.exe -u h
UPTIME OK 10h | 'uptime'=37729s;;;0;

C:\Users\michi\source\repos\icinga2\Bin\Debug\Debug>check_uptime.exe -u m
UPTIME OK 629m | 'uptime'=37740s;;;0;

C:\Users\michi\source\repos\icinga2\Bin\Debug\Debug>check_uptime.exe -u s
UPTIME OK 37749s | 'uptime'=37749s;;;0;

C:\Users\michi\source\repos\icinga2\Bin\Debug\Debug>check_uptime.exe -u ms
UPTIME OK 37755218ms | 'uptime'=37755s;;;0;
```
